runtime/j9vm/jvm.c

```diff
@@ -5911,7 +5922,15 @@ JVM_Sleep(JNIEnv* env, jclass thread, jlong timeout)
 {
 	Trc_SC_Sleep_Entry(env, thread, timeout);
 
+#if JAVA_SPEC_VERSION < 21
 	(*env)->CallStaticVoidMethod(env, jlThread, sleepMID, timeout);
+#else /* JAVA_SPEC_VERSION < 21 */
+	{
+		jlong millis = timeout / 1000000;
+		jint nanos = (jint)timeout % 1000000;
```

Comment:
Parentheses are needed here (to get the right answer):
```c
		jint nanos = (jint)(timeout % 1000000);
```

Reply:
Updated.